### ISSUE TITLE
Illegal instruction: 4 (in target '<my_project>' from project '<my_project>') in release scheme.

### ISSUE THREAD

**Issue by wabisabimori** [AudioKit/AudioKit#2285]
Hi, I encountered the error "illegal instruction: 4" when building my Swift app in release scheme.
The error disappears when I remove the pod of AudioKit and the code related to AudioKit.

Also, in debug scheme, I encountered another error like below.

"in /Users/<my_name>/Library/Developer/Xcode/DerivedData/<my_project>/Build/Products/Debug-iphoneos/AudioKit.framework/AudioKit(EZAudio.o), building for iOS, but linking in object file (/Users/<my_name>/Library/Developer/Xcode/DerivedData/<my_project>/Build/Products/Debug-iphoneos/AudioKit.framework/AudioKit(EZAudio.o)) built for iOS Simulator, file '/Users/<my_name>/Library/Developer/Xcode/DerivedData/<my_project>/Build/Products/Debug-iphoneos/AudioKit.framework/AudioKit' for architecture arm64"

Does anyone have the same issue? 
If you have any ideas which solve this, I would appreciate it so much.
Thanks in advance.

**Comment by wabisabimori** [comment]
For supplement, I'm using Xcode 12.0 and AudioKit 4.11.2.
When I was using Xcode11.x, I'd never had such errors.

**Comment by megastep** [comment]
Are you using CocoaPods? If so you need to update to the latest pre-release version first.

**Comment by wabisabimori** [comment]
Thanks!
@megastep 's advice worked well with some additional fixes. 

Notes on what I did:
1. Install CocoaPods pre-release version on my Mac. 

`sudo gem install cocoapods --pre -n /usr/local/bin`

2. Removed the DerivedData.

`rm -rf ~/Library/Developer/Xcode/DerivedData/`

3. Clean the build folder on Xcode.

`Command + Shift + K`

After that, I encountered another error with release scheme, which says:

"Segmentation fault: 11 (in target '<my_project>' from project '<my_project>')"

So I changed the Swift Compiler Optimization settings to "No Optimization" in TARGET > Build Settings > Optimization Level. 

cf. [ios - Command failed due to signal: Segmentation fault: 11 while archiving - Stack Overflow](https://stackoverflow.com/questions/42167263/command-failed-due-to-signal-segmentation-fault-11-while-archiving/42168123#42168123)

Finally, all errors cleared, and my app has been successfully built. Thanks!
